README.md

```diff
@@ -1,2 +1,84 @@
-# mystica
-**M**ulti bod**Y** **S**imula**T**or max**I**mal **C**oordin**A**tes	
+<h1 align="left">mystica</h1>
+
+<p align="left">
+   <a href="https://github.com/ami-iit/mystica/blob/master/LICENSE"><img src="https://img.shields.io/github/license/ami-iit/mystica" alt="Size" class="center"/></a>
+</p>
+
+**Multi bodY SimulaTor maxImal CoordinAtes**
+
+**mystica** is a library of classes and functions to simulate multibody systems.
+**mystica** computes the system evolution modeling the problem using a state definition that we define _maximal_.
+The _maximal_ representation consists of a set of _non-minimum_ variables $\textbf{q}$ complemented by holonomic constraint $g(\textbf{q})=0$.
+
+---
+
+<p align="center">
+  <b>:warning: REPOSITORY UNDER DEVELOPMENT :warning:</b>
+  <br>The libraries implemented in this repository are still experimental and we cannot guarantee stable API
+</p>
+
+---
+
+### Table of content
+
+- [:hammer: Dependencies](#hammer-dependencies)
+- [:floppy_disk: Installation](#floppy_disk-installation)
+- [:rocket: Usage](#rocket-usage)
+- [:gear: Contributing](#gear-contributing)
+
+
+## :hammer: Dependencies
+
+- [`matlab`](https://mathworks.com/)
+
+Other requisites are:
+
+- [`casadi`](https://web.casadi.org/)
+- [`yamlmatlab`](https://github.com/ewiger/yamlmatlab)
+
+Both [`casadi`](https://web.casadi.org/) and [`yamlmatlab`](https://github.com/ewiger/yamlmatlab) are downloaded and configured in section [Installation](#floppy_disk-installation).
+
+## :floppy_disk: Installation
+
+1. Clone the repo:
+
+``` cmd
+git clone https://github.com/ami-iit/mystica.git
+cd mystica
+```
+
+2. Run in **matlab** the function [`install()`](install.m).
+``` matlab
+install()
+```
+The function [`install()`](install.m) downloads [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge). [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge) is a package manager that downloads and configures our dependencies in conda enviroment called `mystica`.\
+If you already have [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge) configured, you can call [`install()`](install.m) function defining `mambaforge_prefix` value:
```

Comment:
Personal taste/nitpick: I would stick to the simplest installation option in the README, and leave the installation for people that already have mambaforge installed in another page to keep the README as simple as possible.

Reply:
@traversaro 
What do you think of using a [`disclosure widget`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details?retiredLocale=it) to hide the content?
https://github.com/ami-iit/mystica/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R55-R59

Reply:
Good idea.